Code/Mantid/Framework/API/test/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 if ( CXXTEST_FOUND )
   include_directories ( SYSTEM ${CXXTEST_INCLUDE_DIR} ${GMOCK_INCLUDE_DIR} ${GTEST_INCLUDE_DIR} )
 
-  include_directories ( ../../TestHelpers/inc )
+  include_directories ( ../../TestHelpers/inc ../../DataObjects/inc )
```

Comment:
We shouldn't do this. We should be able to build up to API tests at least without any other plugins.

If you need a concrete workspace type then we should create a fake table workspace like is done in `Code/Mantid/Framework/TestHelpers/inc/MantidTestHelpers/FakeObjects.h` with `WorkspaceTester`